cmd/release.go

```diff
@@ -80,9 +81,30 @@ func newCmdRelease() *cobra.Command {
 				}
 			}
 
-			discoveryData, err := getDiscoveryData(projectDir)
-			if err != nil {
-				shared.Logger.Println(styles.Errorf("%s Error: %v", emoji.ErrorExclamation, err))
+			var discoveryData *sharedTypes.DiscoveryData
+			discoveryPath := filepath.Join(projectDir, "Discovery.md")
+			if _, err := os.Stat(discoveryPath); err == nil {
+				discoveryData, err = getDiscoveryData(projectDir)
+				if err != nil {
+					os.Exit(1)
+				}
+			} else if os.IsNotExist(err) {
+				if !shared.IsOutputInteractive() {
+					shared.Logger.Printf("%s Error: %s", emoji.ErrorExclamation, "Discovery.md file is required in non-interactive mode")
```

Comment:
The Discovery file is not required, so in a no interactive terminal we should just skip the prompting instead of throwing an error